tensorflow/examples/android/src/org/tensorflow/demo/env/ImageUtils.java

```diff
@@ -98,6 +98,60 @@ public static void saveBitmap(final Bitmap bitmap, final String filename) {
   // Always prefer the native implementation if available.
   private static boolean useNativeConversion = true;
 
+  public static void convertYUV420SPToARGB8888(
+      byte[] input,
+      int[] output,
```

Comment:
for maximum parallelism with `convertYUV420ToARGB8888`, `output` should be the last argument